src/main/java/uk/gov/pay/directdebit/payments/dao/mapper/GoCardlessEventMapper.java

```diff
@@ -19,6 +19,21 @@
     private static final String RESOURCE_TYPE_COLUMN = "resource_type";
     private static final String JSON_COLUMN = "json";
     private static final String CREATED_AT_COLUMN = "created_at";
+    private static final String DETAILS_CAUSE_COLUMN = "details_cause";
```

Comment:
Looks like all of these constants are only ever used within this file and in a single location so I would be inclined to in-line them where they are being used for easier readability.